content/2022/en/LCNC-SEC-08-Data-and-Secret-Handling-Failures.md

```diff
@@ -11,6 +11,10 @@ title: "LCNC-SEC-08: Data and Secret Handling Failures"
 | --- | --- | --- | --- |
 | 3 | 2 | 3 | 3 |
 
+All applications store or use data, and some of that data is more sensitive than others. If a developer is unaware of what data is considered “sensitive data”, or the processes to protect that data, the sensitive information could be exposed.   
```

Comment:
Most rather than all?

Reply:
Thank you! Updated: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/pull/56/commits/9d2c6ff3715fc8135f8f902756bc3d2e18777780